tests/testthat/test-geom-dotplot.R

```diff
@@ -125,6 +125,34 @@ test_that("geom_dotplot draws correctly", {
     ggplot(dat, aes(x)) + geom_dotplot(binwidth = .4, stackdir = "centerwhole") + coord_flip()
   )
 
+  # Stacking methods with stackratio < 1
```

Comment:
Can you please make this a separate test? (The existing huge test unfortunately isn't a good pattern to follow). And can you please also create the smallest possible dataset needed to demonstrate the problem?

Reply:
Sure, done --- I pulled out the four tests for `stackratio = 1.5` into a separate test with a single doppelganger on a simplified dataset that should be sensitive to problems with any of the four values of `stackdir`. Dunno if you would rather those be four separate small doppelgangers or one merged doppelganger as I have done.